docs/association/security/an_bug_bounty.md

```diff
@@ -43,6 +43,13 @@ In scope for the bug bounty are all the smart contract components of the Aragon
         - Excluding `contracts/test/`
         - Excluding `contracts/standards/`
 
+!!! abstract "**Aragon Network fundraising** ([v1](https://github.com/aragonone/fundraising/tree/v1.0.0) and [future minor versions](https://github.com/aragonone/fundraising/releases))"
+    Smart contracts for the Aragon Network fundraising process
```

Comment:
```suggestion
    Smart contracts for the Aragon Network Juror (ANJ) token bonding curve
```

Reply:
Modified slightly because it's not really a fundraising campaign, it's a token exchange using a bonding curve.